src/Devices/EthernetServer_TC.cpp

```diff
@@ -3,6 +3,8 @@
 #include <avr/wdt.h>
 
 #include "DateTime_TC.h"
+#include "Devices/Ethernet_TC.h"
+#include "Devices/LiquidCrystal_TC.h"
```

Comment:
Why was LiquidCrystal_TC.h added?

Reply:
I closed this yesterday due to these problems. Check the open one.